tests/host_callback_test.py

```diff
@@ -103,7 +100,7 @@ def repl_func(match_group):
     else:
       return "..."
   what = re.sub(r"tap_func_=(.*)", repl_func, what)
-  tst.assertMultiLineStrippedEqual(expected, what)
+  tst.assertMultiLineStrippedEqual(textwrap.dedent(expected), what)
```

Comment:
The `assertMultiLineStrippedEqual` already ignores white space; perhaps `dedent` can be part of its implementation?

Reply:
Good call, I moved that part upstream.